src/cpl/nuopc/rof_import_export.F90

```diff
@@ -0,0 +1,679 @@
+module rof_import_export
```

Comment:
It seems like this is really pretty generic in terms of component. Could some of this go under cime?

Reply:
No - this should not go under CIME. NUOPC components should be able to be plugged into any system and therefore the code to do that should not have use statements into CIME share code. Since the target coupled system that uses that code might not be CIME compliant (as is the case with UFS).